lib/flow_runner/custom_blocks/dynamic_select_one_response.ex

```diff
@@ -36,6 +36,17 @@ defmodule FlowRunner.CustomBlocks.DynamicSelectOneResponse do
         context.vars,
         FlowRunner.expression_callbacks_module()
       )
+      |> Enum.map(fn
+        # Handle the specific case of a two-element list where options are time formatted
+        # i.e. list("cta", NextCard, map(times_formatted_options_list, &[&1,&1]))
+        [option_a, _option_b] when is_struct(option_a, Time) ->
+          time_string = Time.to_string(option_a)
+          [time_string, time_string]
```

Comment:
Can we please add a unit test for this case? It's not being covered, I used this to check
```bash
MIX_ENV=test mix coveralls.html
open cover/excoveralls.html
```

<img width="741" alt="Screenshot 2025-05-11 at 23 49 28" src="https://github.com/user-attachments/assets/3a5380c8-f525-448c-8099-64c6fd83ca80" />

Reply:
Great! Thanks @davidkhk!